src/starknetCli.ts

```diff
@@ -29,14 +29,7 @@ const NETWORK = 'network';
 const WALLET = 'wallet';
 
 const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', 'warp_venv', 'bin')}:$PATH`;
-const CAIRO1_COMPILE_BIN = path.resolve(
-  __dirname,
-  '..',
-  'cairo1',
-  getPlatform(),
-  'bin',
-  'starknet-compile',
-);
+const CAIRO1_COMPILE_BIN = path.resolve(__dirname, '..', 'cairo1', getPlatform(), 'bin', 'warp');
```

Comment:
We could have BASE_PATH to warp directory as  const and then just do `WARPLIB_PATH = path.join(BASE_PATH, 'warplib', 'src')`, same with venv and compile bin